sertmer/swapi-trivia
# The Motherload of commits

#### What’s this PR do?
This PR is kind of my own fault. I kept on doing things without committing so bear with me. It fetch's all of the Data in a displayable format. CharacterContainer / CharacterCard files created, structured, and displayed properly. fetchAPI's file created that isolates all fetch's for easier calling.
Allows character cards to be appended specific to the movie that was clicked. Added minimal styling to allow for visual testing. 

#### Where should the reviewer start?
CharacterContainer.js 
CharacterCard.js 
fetchAPIs.js
respective css files if you're really interested. 

#### How should this be manually tested?
Wait 4-5 minutes for page to resolve the FAT fetch. Click view on a film to see accurately appended character cards. 

#### Any background context you want to provide?
I was on a roll and couldn't stop myself. I apologize in advance.
